src/app/app.constants.ts

```diff
@@ -205,6 +205,8 @@ const appHeaderTitle = {name: 'Manage Cases', url: '/'};
 // Making the base URL for case details a constant.
 const caseDetailsUrl: string = '/cases/case-details/';
 
+const serviceMessagesFeatureToggleKey: string = 'mc-service-messages';
+const serviceMessageCookie = 'service_messages';
```

Comment:
@paulhowes-HMCTS - very minor point (probably a bit pedantic) but for consistency with the other `const`s, you may wish to declare the variable type as `string`

Reply:
Actually, I thought this would _also_ fail linting as it's considered "trivial" by the linter.

Reply:
Going to add the : string in there, and Paul, it's there to be consistent with the rest of the code in the file.